envcfg.go

```diff
@@ -303,6 +303,28 @@ func (ec *Envcfg) MustInt(key string) int {
 	return i
 }
 
+// KeyExist checks if the key exists in the section, if it is found in the
+// section it returns true, else false 
+func (ec *Envcfg) KeyExist(sectionNameAndKey string) bool {
+	name, key := ec.config.NameSplitFunc(sectionNameAndKey)
+	section := ec.config.GetSection(name)
+	if section == nil {
+		return false
+	}
+
+	sectionKeys := section.Keys()
+	if len(sectionKeys) == 0 {
+		return false 
+	}
+
+	for _, sKey := range sectionKeys {
+		if strings.ToLower(key) == sKey {
+			return true
+		}
+	}
+	return false 
+}
```

Comment:
Can we extend this to support the checking of the key regardless of the hierarchy where it was declared?
``` 
[key0]

[key1]
value1=""

[key2 "subkey0"]

[key3 "subkey0"]
value1=""
```
```
KeyExist("key0")  # true
KeyExist("key1.value1")  # true
KeyExist("key2.subkey0")  # true
KeyExist("key3.subkey0.value1")  # true
```

Reply:
@grikwong i think key* is called a section in this context, so to be able to check a section and a key?